src/Components/Search/SearchBar.tsx

```diff
@@ -190,14 +190,7 @@ export class SearchBar extends Component<Props, State> {
   }
 
   onBlur = e => {
-    // This event _also_ fires when a user clicks on a link in the preview pane.
```

Comment:
This is left over from preview pane stuff, was causing a JS crash since `this.containerRef` was `undefined`.